### PR TITLE
Fix plugin main file

### PR DESCRIPTION
**Pull request by szepeviktor**
## Purpose

Please see comments below.

### CODE REVIEW

File: blocks-for-github.php [hunk 1]
```diff
@@ -16,28 +16,26 @@
 
 const BLOCKS_FOR_GITHUB_FILE = __FILE__;
 define( 'BLOCKS_FOR_GITHUB_DIR', plugin_dir_path( BLOCKS_FOR_GITHUB_FILE ) );
-define( "BLOCKS_FOR_GITHUB_URL", plugin_dir_url( BLOCKS_FOR_GITHUB_FILE ) );
+define( 'BLOCKS_FOR_GITHUB_URL', plugin_dir_url( BLOCKS_FOR_GITHUB_FILE ) );
 const BLOCKS_FOR_GITHUB_SCRIPT_ASSET_PATH = BLOCKS_FOR_GITHUB_DIR . '/build/index.asset.php';
-define( 'BLOCKS_FOR_GITHUB_SCRIPT_ASSET', require( BLOCKS_FOR_GITHUB_SCRIPT_ASSET_PATH ) );
```
Comment (szepeviktor) [inline blocks-for-github.php:21]: `require` is a statement, not a function.

File: blocks-for-github.php [hunk 2]
```diff
@@ -16,28 +16,26 @@
...
+define( 'BLOCKS_FOR_GITHUB_SCRIPT_ASSET', require BLOCKS_FOR_GITHUB_SCRIPT_ASSET_PATH );
 const BLOCKS_FOR_GITHUB_SCRIPT_NAME = 'blocks-for-github-script';
 
 /**
- * Require WP version 5.5+
+ * Require WP version 6.5+
```
Comment (szepeviktor) [inline blocks-for-github.php:25]: Meta header says 6.5

File: blocks-for-github.php [hunk 3]
```diff
@@ -16,28 +16,26 @@
...
             wp_die(
-                esc_html__( 'Blocks for GitHub requires WordPress version 5.5 or greater.', 'blocks-for-github' ),
+                esc_html__( 'Blocks for GitHub requires WordPress version 6.5 or greater.', 'blocks-for-github' ),
                 esc_html__( 'Error Activating', 'blocks-for-github' )
             );
         }
     }
 );
 
+require_once __DIR__ . '/vendor/autoload.php';
```
Comment (szepeviktor) [inline blocks-for-github.php:39]: Using absolute path is robust.

File: blocks-for-github.php [hunk 4]
```diff
@@ -16,28 +16,26 @@
...
 
-require_once 'vendor/autoload.php';
-
-$bootstrapPlugin = new Bootstrap();
-$bootstrapPlugin->init();
+(new Bootstrap())->init();
```
Comment (szepeviktor) [inline blocks-for-github.php:41]: Remove this global variable.

**Comment by DevinWalker** [comment]
@szepeviktor thank you for your contributions. Excellent code review :)

**Pull request merged**